_posts/2019-12-13-green-sock-animation.md

```diff
@@ -0,0 +1,114 @@
+---
+title:  "Animation Beyond the Canvas"
+date: 2019-12-13 00:00:00 +0800
+author: jimmy
+---
+
+Canvas animations are great for eye catching displays, but what about animations on buttons? Or anything else that isn't a canvas? You _could_ use native css animations, but those can be cumbersome, especially for complex animations. Instead, we'll be taking a look at the popular animation library GreenSock.
```

Comment:
you link to Greensock later, but in this opening summary would be nice to link Greensock here as well I suppose?